man/systemd-creds.xml

```diff
@@ -25,6 +25,7 @@
     <cmdsynopsis>
       <command>systemd-creds</command>
       <arg choice="opt" rep="repeat">OPTIONS</arg>
+      <arg choice="plain">COMMAND</arg>
```

Comment:
I think it would be nice to add `[ARGS]` or `[ARGUMENTS]` after `COMMAND`.

Reply:
Done.